docs/develop/nova/block_explorer.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Autonomys block explorer
+title: Astral Block Explorer
```

Comment:
We don't really reference Astral anywhere on this page, so we can either keep "Autonomys Block Explorer", or adjust it to "Nova Block Explorer"